awx/main/utils/formatters.py

```diff
@@ -21,7 +24,81 @@ def format(self, record):
         return logging.Formatter.format(self, record)
 
 
-class LogstashFormatter(LogstashFormatterVersion1):
+class LogstashFormatterBase(logging.Formatter):
```

Comment:
@wenottingham I assume MIT means this is fine?

https://github.com/ansible/awx/blob/devel/docs/licenses/python-logstash.txt

Reply:
ping @wenottingham, are there are any special doc/licensing considerations to make here if we're bundling this code from this external library?

Reply:
Good question, I'd also like @wenottingham to have a quick glance. Otherwise I'm ready to slap the "mergit" on this.

Reply:
by wfm: the way it's denoted there is fine. I haven't tested the code.